.gitignore

```diff
@@ -28,7 +28,10 @@
 *.patch
 *.log
 /doxygen
-.vs/
+.*
+configure
+autom4te.cache
+simres.d.vs/
```

Comment:
Please do not include the change which is not related to the main topic of this PR.